### ISSUE TITLE
MatplotlibDeprecationWarnings in tests

### ISSUE THREAD

**Issue by texadactyl** [UCBerkeleySETI/blimpy#140]
**Describe the bug**
During a pytest run, one can see these non-fatal warnings from test_plotting_doesnt_cause_exceptions() in tests/test_filterbank_voyager.py:
```
MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
```

**To Reproduce**
1. Install blimpy
2. Run ```pytest```
3. See error

**Expected behavior**
No matplotlib warnings should appear.

**Setup**
 - Python version: 3.8.3
 - Blimpy version: latest from github

**Fix**
In test_plotting_doesnt_cause_exceptions(), insert this between the H5 and FIL tests:
```
plt.clf()
```
In other words, stop the FIL tests from reusing the axes created during the H5 tests.

**Comment by telegraphic** [comment]
Nice straightforward fix -- looks like this one is closed now too?
